diff --git a/node_modules/xstate/es/devTools.js b/node_modules/xstate/es/devTools.js
index 8c3e390..c75029b 100644
--- a/node_modules/xstate/es/devTools.js
+++ b/node_modules/xstate/es/devTools.js
@@ -27,7 +27,7 @@ function getDevTools() {
 }
 
 function registerService(service) {
-  if (IS_PRODUCTION || !getGlobal()) {
+  if (!getGlobal()) {
     return;
   }
 
diff --git a/node_modules/xstate/lib/devTools.js b/node_modules/xstate/lib/devTools.js
index 628aa04..d9df632 100644
--- a/node_modules/xstate/lib/devTools.js
+++ b/node_modules/xstate/lib/devTools.js
@@ -24,7 +24,7 @@ function getDevTools() {
     return undefined;
 }
 function registerService(service) {
-    if (environment_1.IS_PRODUCTION || !getGlobal()) {
+    if (!getGlobal()) {
         return;
     }
     var devTools = getDevTools();
